packages/commands/generate/src/index.ts

```diff
@@ -7,18 +7,20 @@ import { JsonFileLoader } from '@graphql-toolkit/json-file-loader';
 import { UrlLoader } from '@graphql-toolkit/url-loader';
 import { GitLoader } from '@graphql-toolkit/git-loader';
 import { GithubLoader } from '@graphql-toolkit/github-loader';
-import { GraphQLBackendCreator, GraphQLGeneratorConfig, Client, IGraphQLBackend, DatabaseSchemaManager } from 'graphback';
+import { GraphQLBackendCreator, GraphbackGeneratorConfig, IGraphQLBackend, DatabaseSchemaManager, ClientDocuments, UpdateDatabaseIfChanges, migrate } from 'graphback';
 import { ensureFile } from 'fs-extra';
 import { writeFile as fsWriteFile } from 'fs';
 import { join } from 'path';
+
 export interface GenerateConfig {
   folders: {
     model: string;
     schema: string;
     resolvers: string;
     client: string;
+    migrations: string;
   };
-  graphqlCRUD: GraphQLGeneratorConfig;
+  graphqlCRUD: GraphbackGeneratorConfig;
   db: { database: string; dbConfig: any; };
 }
```

Comment:
We can use YAML Schema validator in here to make sure the structure of config is correct regarding to the subcommand of `generate`.

Reply:
I'm hoping for all of this to go away when we move to codegen - then file locations can be removed etc.